framework/tests/value/Issue3001.java

```diff
@@ -0,0 +1,7 @@
+class Issue3001 {
```

Comment:
As there is nothing value-checker specific in this test, would it make sense to move it to `framework/tests/all-systems`?